crates/gradbench/src/cli.rs

```diff
@@ -75,6 +76,10 @@ enum Commands {
         /// A path to save the full log. For example: `log.jsonl`
         #[clap(short, long)]
         output: Option<PathBuf>,
+
+        /// The timeout in seconds for tool responses.
+        #[clap(long, default_value_t = 3600)]
```

Comment:
Nice, somehow I didn't know about [`default_value_t`](https://docs.rs/clap/4.5.23/clap/_derive/index.html#arg-attributes). For consistency I'll also go do this on the other args that have defaults.